app.rb

```diff
@@ -1,122 +1,177 @@
-require_relative 'book'
 require_relative 'person'
+require_relative 'classroom'
 require_relative 'student'
 require_relative 'teacher'
+require_relative 'book'
 require_relative 'rental'
-require_relative 'classroom'
 
 class App
-  attr_accessor :books, :people, :rentals
-
   def initialize
-    @books = []
     @people = []
+    @books = []
     @rentals = []
   end
 
+  def display_menu
+    puts 'Welcome to the School Library App'
+    puts 'Please choose an option by entering a number:'
+    puts '1. List all books'
+    puts '2. List all people'
+    puts '3. Create a person'
+    puts '4. Create a book'
+    puts '5. Create a rental'
+    puts '6. List all rentals for a given person id'
+    puts '7. Exit'
```

Comment:
- [ ] Hey there! 🌟 Following the feedback from previous reviewers, it's important to note that all requests for user inputs or validations should be housed in the main.rb file or within classes/methods specifically designated for handling user inputs. 🛠️ As of now, this adjustment is still pending in your refactored code. Please make sure to implement this for a more organized and structured codebase. If you have any questions or need further clarification, feel free to ask! 🚀